src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedToObject = {};
+
+  const arrayOfProperties = sourceString.split(';').map(string => {
+    return string.trim().split(':');
+  });
```

Comment:
```suggestion
  const arrayOfProperties = sourceString
    .split(';')
    .map(string => string.trim().split(':'));
```
Старайся переносить методы на новую строку, а начинку - наоборот, по возможности не переносить. Код будет выглядеть читаемей

Reply:
Принято 👍🏻